tests/test_github.py

```diff
@@ -1260,6 +1316,15 @@ def test_fetch_pull_enterprise(self):
                                    'Link': '<' + GITHUB_ENTERPRISE_ISSUES_URL + '/?&page=2>; rel="next", <' +
                                            GITHUB_ENTERPRISE_ISSUES_URL + '/?&page=2>; rel="last"'
                                })
+
+        httpretty.register_uri(httpretty.GET,
```

Comment:
In the test `test_fetch_pull_enterprise` it seems that no assertion is done over the commit comments data

Reply:
Sure, will add it ;)